website/content/en/docs/cli/operator-sdk_init.md

```diff
@@ -28,30 +28,30 @@ operator-sdk init [flags]
 
 ```
   # Scaffold a project using the apache2 license with "The Kubernetes authors" as owners
-  operator-sdk init --project-version=2 --domain example.org --license apache2 --owner "The Kubernetes authors"
+  operator-sdk init --project-version=3-alpha --domain example.org --license apache2 --owner "The Kubernetes authors"
```

Comment:
This may be outdated documentation upstream in kubebuilder as 3-alpha is no longer a project version.